test/mod/test_util_mnt.py

```diff
@@ -21,6 +21,14 @@ def test_mount_guard_failure_msg(tmp_path):
     assert "special device /dev/invalid-src does not exist" in str(e.value)
 
 
+@pytest.mark.skipif(os.getuid() != 0, reason="root only")
```

Comment:
super-nipitck: I don't think this needs to be guarded by root only because the code that needs root (i.e. to mount) will never be hit, it errors before already

Reply:
Ah that's a fair point, I guess we could fix it up if we cleanup the rest of the mount stuff maybe?